featurebyte/tile/tile_cache.py

```diff
@@ -482,7 +482,7 @@ async def _get_unique_tile_infos(
                     out[TileInfoKey.from_tile_info(info)] = info
             if i % 10 == 0 and progress_callback is not None:
                 await progress_callback(
-                    int(i + 1 / len(nodes) * 100), "Checking tile cache availability"
+                    int((i + 1) / len(nodes) * 100), "Checking tile cache availability"
```

Comment:
If these values exceed 100, it will throw pydantic error:
```
pydantic.error_wrappers.ValidationError: 1 validation error for ProgressModel
percent
  ensure this value is less than or equal to 100 (type=value_error.number.not_le; limit_value=100)
```

Reply:
Would it be better to add clipping in the callback function for more general handling?